open_academy/wizards/attendee_wizards.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class Attendee(models.TransientModel):
```

Comment:
```suggestion
class AttendeeWizard(models.TransientModel):
```
Change all it's references

Reply:
Done